Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -779,13 +779,39 @@ public struct RenderNodeTranslator: SemanticVisitor {
             
             return seeAlsoSections
         } ?? .init(defaultValue: [])
+
+        if let callToAction = article.metadata?.callToAction {
```

Comment:
Will this work for symbol pages too? (That can probably be left for a future PR if not, just wondering. I think we already have the limitation where `@Metadata` is only supported in doc extension files fwiw.)

Reply:
Not with the current implementation, but it's not that much more work to add it there. (I would need to add this block to `visitSymbol()` as well for it to be picked up on symbol pages.)

Reply:
If it's small I think it might be worth doing here just to reduce confusion – otherwise we can open a bug for it so that folks are aware.

Reply:
Adding it to symbol pages turned out to be a bit more complicated than i thought, so i'm going to punt on it for now.

Reply:
Sounds good 👍🏻 I think a GitHub Issue to track this would be helpful so we don't lose track.

Reply:
I've filed https://github.com/apple/swift-docc/issues/436 to track this.